docs/guides/classical-feedforward-and-control-flow.ipynb

```diff
@@ -202,196 +192,218 @@
-    "\n",
-    "In the example below, we apply Hadamard gates to two qubits and measure them. Then, we create a while loop that repeats this procedure while the measurement outcome is 11. As a result, the final measurement should never be 11, with the remaining possibilities appearing with approximately equal frequency."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 7,
-   "id": "95795aa7-9d4d-4c35-ba0e-48e6ba4f41b9",
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Image src=\"/docs/images/guides/classical-feedforward-and-control-flow/extracted-outputs/95795aa7-9d4d-4c35-ba0e-48e6ba4f41b9-0.svg\" alt=\"Output of the previous code cell\" />"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "qubits = QuantumRegister(2)\n",
-    "clbits = ClassicalRegister(2)\n",
-    "circuit = QuantumCircuit(qubits, clbits)\n",
-    "\n",
-    "q0, q1 = qubits\n",
-    "c0, c1 = clbits\n",
-    "\n",
-    "circuit.h([q0, q1])\n",
-    "circuit.measure(q0, c0)\n",
-    "circuit.measure(q1, c1)\n",
-    "with circuit.while_loop((clbits, 0b11)):\n",
-    "    circuit.h([q0, q1])\n",
-    "    circuit.measure(q0, c0)\n",
-    "    circuit.measure(q1, c1)\n",
-    "\n",
-    "circuit.draw(\"mpl\")\n",
-    "\n",
-    "# example output counts: {'01': 334, '10': 368, '00': 322}"
+    "## Qiskit Runtime Limitations\n",
+    "\n",
+    "Be aware of the following constraints when running dynamic circuits in Qiskit Runtime.  Note that as new features and improvements are added throughout the coming year, some of these limitations will be lifted.\n",
+    "\n",
+    "- Due to the limited physical memory on control electronics, there is also a limit on the number of if statements and broadcasted bits. This limit is a function of the number of broadcasts and the number of broadcasted bit in a job (not a circuit).\n",
+    "\n",
+    "   When processing an if condition, measurement data needs to be transferred to the control logic to make that evaluation. A broadcast is a transfer of unique classical data, and broadcasted bits is the number of classical bits being transferred. Consider the following:\n",
+    "\n",
+    "   ```python\n",
+    "   c0 = ClassicalRegister(3)\n",
+    "   c1 = ClassicalRegister(5)\n",
+    "   ...\n",
+    "   with circuit.if_test((c0, 1)) ...\n",
+    "   with circuit.if_test((c0, 3)) ...\n",
+    "   with circuit.if_test((c1[2], 1)) ...\n",
+    "   ```\n",
+    "   In the previous code example, the first two `if_test`s on `c0` are considered one broadcast because the content of `c0` did not change, and thus does not need to be re-broadcasted. The `if_test` on `c1` is a second broadcast. The first one broadcasted all three bits in `c0` and the second broadcasted just oine bit, making a total of four broadcasted bits.\n",
```

Comment:
```suggestion
    "   In the previous code example, the first two `if_test`s on `c0` are considered one broadcast because the content of `c0` did not change, and thus does not need to be re-broadcasted. The `if_test` on `c1` is a second broadcast. The first one broadcasted all three bits in `c0` and the second broadcasted just one bit, making a total of four broadcasted bits.\n",
```